server/models/blog.js

```diff
@@ -0,0 +1,39 @@
+const { db } = require ('../db/config')
+const SQL = require("sql-template-strings");
+
+class Blog {
+    constructor(data){
+        this.id = data.id
+        this.title = data.title
+        this.author = data.author
+        this.content = data.content
+    }
+
```

Comment:
We noticed you guys don't have get all method, we don't think it's neccessary but it was useful to us to keep tabs on whether the server was functioning properly at all

Reply:
Good point, will implement one!